cache.h

```diff
@@ -25,20 +25,21 @@ typedef std::array<float, 4> MaterialsWeightsArray;
 
 class Heightfield {
 public:
-    float heightField;
+    float height;
 
     std::array<unsigned char, 4> biomesVectorField;
     std::array<unsigned char, 4> biomesWeightsVectorField;
 
-    float waterFactor;
+    float waterHeight = MIN_WORLD_HEIGHT;
+    float waterFactor = 0.f;
 
     vm::vec3 normal{0.f, 1.f, 0.f};
 
     MaterialsArray materials;
     MaterialsWeightsArray materialsWeights;
 
     float getHeight() const {
-      return heightField;
+      return height;

```

Comment:
Thanks! I know it's a small change, but this is exactly the kind of thing that makes code more readable and easier to maintain. I was definitely confused by this when I first read the code- `return heightfield` seems to imply that we're producing the entire field of values, instead of a single height.

This is much better, thanks!